lib/core/presentation/pages/home/home_page.dart

```diff
@@ -79,19 +88,31 @@ class _HomePageState extends State<HomePage> {
           ],
         ),
         backgroundColor: LemonColor.black,
-        body: SwipeDetector(
-          child: const HomeNewsfeedListView(),
-          onSwipeUp: () {},
-          onSwipeDown: () {},
-          onSwipeLeft: () {
-            context.read<AuthBloc>().state.maybeWhen(
-                  authenticated: (session) =>
-                      AutoRouter.of(context).navigate(const ChatListRoute()),
-                  orElse: () =>
-                      AutoRouter.of(context).navigate(const LoginRoute()),
-                );
-          },
-          onSwipeRight: () {},
+        body: UpgradeAlert(
```

Comment:
should move this logic to root page, home_page should not care about these logic